specs/core/0_beacon-chain.md

```diff
@@ -1140,15 +1140,15 @@ def slash_validator(state: BeaconState,
     current_epoch = get_current_epoch(state)
     initiate_validator_exit(state, slashed_index)
     state.validator_registry[slashed_index].slashed = True
-    state.validator_registry[slashed_index].withdrawable_epoch = current_epoch + LATEST_SLASHED_EXIT_LENGTH
+    state.validator_registry[slashed_index].withdrawable_epoch = Epoch(current_epoch + LATEST_SLASHED_EXIT_LENGTH)
```

Comment:
If `current_epoch` and `LATEST_SLASHED_EXIT_LENGTH` are both of type `Epoch` do we need to wrap the sum with `Epoch( ... )`?

Reply:
`Epoch` is defined as "EpochNumber/EpochID" and `LATEST_SLASHED_EXIT_LENGTH` is defined as "diff between Epochs".

I'm slightly in favor of the status quo but I'm open to it; for `Epoch` and `Slot`, we can change it to the more flexible definition if we want. :)